### ISSUE TITLE
not generalizing well

### ISSUE THREAD

**Issue by microprediction** [microprediction/sklearned#1]
Perhaps it makes sense for the purpose of this ticket to fix an experiment, such as [this one](https://github.com/microprediction/sklearned/blob/main/experiments_on_skaters/nevergrad_oneplus--keras_mostly_linear_27--quick_aggressive_ema_ensemble--3--160.py)

If you trace in you'll see how it is generating surrogate training data. This could be improved or blown away I'm not sure. But maybe we first just ensure that the normalization is consistent here and where it is used in a [successor skater](https://github.com/microprediction/successor/blob/main/successor/skaters/scalarskaters/scalarsimpleskaters.py)

Some possibilities: 

- The training and prediction data should not overlap at all. Currently they do. 
- Different scaling of the input data to be more consistent
- Different data augmentation is required
- Bug in interpolation in k ? 
- Just not enough training data
 
Do we backtrack to something even simpler like a fast moving average? Maybe!
